spec/ruby/core/time/succ_spec.rb

```diff
@@ -2,31 +2,33 @@
 require_relative 'fixtures/classes'
 
 describe "Time#succ" do
-  it "returns a new time one second later than time" do
-    suppress_warning {
-      @result = Time.at(100).succ
-    }
+  ruby_version_is ""..."2.6" do
```

Comment:
What happened to ruby 2.7?

Reply:
Indeed 2.7 should be included, I'll fix this